resources/[system]/sessionmanager/server/host_lock.lua

```diff
@@ -53,7 +53,7 @@ end)
 AddEventHandler('hostedSession', function()
     -- check if the client is the original locker
     if currentHosting ~= source then
-        -- TODO: drop client as they're clearly lying
+        DropPlayer(source, 'Connection to the server was lost')
```

Comment:
I don't think it's actually about "server connection". It's most likely about player tried to host a session but it was hosted before for some reason.

Reply:
Ohh i get it now.
I did not really look into it i just saw a quick easy thing in todo list.